sycl/test-e2e/bindless_images/read_sampled.cpp

```diff
@@ -1,4 +1,8 @@
-// REQUIRES: cuda
+// REQUIRES: aspect-ext_oneapi_bindless_images
+
+// UNSUPPORTED: hip || level_zero
+// UNSUPPORTED-INTENDED: Returning non-FP values from fetching fails on HIP.
```

Comment:
```suggestion
// UNSUPPORTED-INTENDED: Returning non-FP values from sampling fails on HIP.
```
Fetching implies no sampling is occuring.